packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -535,15 +529,25 @@ const VirtualListBaseFactory = (type) => {
 							}
 
 						}
-					} else if (
-						directions.right && repeat ||
-						directions.left && Spotlight.move(direction) ||
-						(directions.up || directions.down) && (repeat || moveFocusStraight({id: this.props.spotlightId, direction}) && currentTarget.contains(Spotlight.getCurrent()))
-					) {
-						ev.preventDefault();
-						ev.stopPropagation();
-					} else if (!repeat) {
-						isLeaving = true;
+					} else {
+						const spottables = Spotlight.getSpottableDescendants(spotlightId);
+						const spottablesTotal = spottables.length;
+						const firstScrollButton = spottables[spottablesTotal - 2];
+						const lastScrollButton = spottables[spottablesTotal - 1];
```

Comment:
if `focusableScrollbar` props is `false`, then the last list item will be assigned to `lastScrollButton`.

Reply:
Good point. I've addressed this by:
- using a `querySelector` to verify scroll buttons, since they will be the only spottable components outside of the list wrapping container
- updating the focus logic to account for `focusableScrollbar`